addons/website/__manifest__.py

```diff
@@ -11,6 +11,7 @@
         'digest',
         'web',
         'web_editor',
+        'html_editor',
```

Comment:
You can squash this second commit into the first one no? It's a test to test your first commit's feature, no?

Reply:
Yes, I squashed them. I didn't do it cause I wasn't sure if we want the test since it uses hoot :pray: